nrfs/include/internal/nrfs_hdr.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef NRFS_HDR_H
+#define NRFS_HDR_H
+
+#include <internal/nrfs_phy.h>
+
+#include <internal/requests/nrfs_temp_reqs.h>
+#include <internal/requests/nrfs_mram_reqs.h>
+#include <internal/requests/nrfs_reset_reqs.h>
+#include <internal/requests/nrfs_usb_reqs.h>
+#include <internal/requests/nrfs_pmic_reqs.h>
+#include <internal/requests/nrfs_dvfs_reqs.h>
+#include <internal/requests/nrfs_diag_reqs.h>
+#include <internal/requests/nrfs_clock_reqs.h>
```

Comment:
What is the benefit of having all the misused enums, mostly just a few lines, scattered throughout all these headers and not having it just here?

Reply:
If we want to reduce the number of files, we can discuss the eventuality of moving those enums here.

Reply:
I propose to change it in the next step (after nrfs publishing).